contracts/Auction.sol

```diff
@@ -123,10 +118,11 @@ contract Auction {
 
     function addTime() public {
```

Comment:
shall we add an argument on how much time we would like to add ?

Reply:
It is possible to do such but I guess its a matter of how we want to use the function. When I was writing the code for this, I don't think we discussed its use enough and from what we discussed, I tailored this function specifically to only trigger within the last few minutes(chose 2 minutes for consistency) and I figured adding a minute every time it was triggered and the condition was satisfied( remaining time <= 2 min) was fair enough.